sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-generation.service.ts

```diff
@@ -621,12 +621,16 @@ export class ECertGenerationService {
     disbursements: ECertDisbursementSchedule[],
   ) {
     for (const disbursement of disbursements) {
-      disbursement.tuitionRemittanceEffectiveAmount =
+      const maxTuitionRemittance =
         this.confirmationOfEnrollmentService.getMaxTuitionRemittance(
           disbursement.disbursementValues,
           disbursement.studentAssessment.application.currentAssessment.offering,
           MaxTuitionRemittanceTypes.Effective,
         );
+      disbursement.tuitionRemittanceEffectiveAmount =
```

Comment:
disbursement.tuitionRemittanceEffectiveAmount = Math.min(
  disbursement.tuitionRemittanceRequestedAmount,
  maxTuitionRemittance
);

Just a suggestion, this can be refactored like his